control-plane-operator/controllers/hostedcontrolplane/v2/kas/config.go

```diff
@@ -412,3 +409,58 @@ func auditWebhookConfigFile() string {
 	cfgDir := kasAuditWebhookConfigFileVolumeMount.Path(ComponentName, auditWebhookConfigFileVolumeName)
 	return path.Join(cfgDir, hyperv1.AuditWebhookKubeconfigKey)
 }
+
+// enforceFeatureGates is a helper function that ensures the feature gates
+// are set to a particular state based on the provided feature gate strings.
+// If the existing set of feature gate strings does not contain a
+// desired feature gate string it is added.
+// If the existing set of feature gate strings does contain a desired feature gate,
+// but sets the state to a different value, it will be overrided to match
+// the desired state.
+func enforceFeatureGates(featureGates []string, addIfMissing ...string) []string {
```

Comment:
nit: addIfMissing seems a bit misleading to me, since if it's not missing is also clobbered

Reply:
Ah, good catch - I missed changing the name of this from the iteration where it was only added if missing :)